.github/workflows/sonar.yml

```
@@ -0,0 +1,19 @@
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        types: [ opened, synchronize, reopened ]
+jobs:
+    sonarqube:
+        name: SonarQube
+        runs-on: ubuntu-latest
+        steps:
+            -   uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)